docs/services/qix-engine/introduction.md

```diff
@@ -67,6 +67,7 @@ Below is a list of Qlik Associative Engine command line parameters used in Qlik
 | ValidateJsonWebTokens | <p>`0` (Not enforced)</p>`1` (Enforce unsigned or signed JWTs)<p>or</p>`2` (Enforce signed JWTs) | Parameter for enforcing validation of JWT. For more details see the [Authorization chapter](../../tutorials/authorization.md). | `0` (Not enforced) |
 | JsonWebTokenSecret | string | Requires the `ValidateJsonWebTokens` parameter to be set. For more details see the [Authorization chapter](../../tutorials/authorization.md). | n/a |
 | JsonWebTokenPath | File path | Path to pem file. For more details see the [Authorization chapter](../../tutorials/authorization.md). | n/a |
+| BuildAppCacheAtStartup | `0` (disabled) or `1` (enabled) | Will make the Qlik Associative Engine build a cache of documents found in the `DocumentDirectory` at startup, making it possible to list them through [getDocList API](./apis/qix/global/#getdoclist) | `0` (disabled) |
```

Comment:
The link seems broken. Points to `/services/qix-engine/introduction/apis/qix/global/#getdoclist` for me.